chromium_src/chrome/installer/DEPS

```diff
@@ -1,4 +1,5 @@
 include_rules = [
+  "+brave/components/brave_vpn",
```

Comment:
this should be limited to a specific target

Reply:
done